### PR TITLE
Solve WTForms 3.0 validators.Required dependency issue

### LINKED ISSUE #75: DeprecationWarning: Required is going away in WTForms 3.0, use DataRequired
Hi,

After updating the Flask / Flask-WTF and WTForms i receive the following warning

```
2019-10-08 15:28:30,129 - INFO -  * Restarting with stat MainThread
/usr/local/lib/python3.7/site-packages/flask_wtf/recaptcha/widgets.py:5: DeprecationWarning: The import 'werkzeug.url_encode' is deprecated and will be removed in Werkzeug 1.0. Use 'from werkzeug.urls import url_encode' instead.
  from werkzeug import url_encode
/usr/local/lib/python3.7/site-packages/flask_ldap3_login/forms.py:26: DeprecationWarning: Required is going away in WTForms 3.0, use DataRequired
  username = wtforms.StringField('Username', validators=[validators.Required()])
```

If you think to fix it and to release a new build.

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #75. Same changes suggested by @HeMan. Tested locally and it works fine!

### CODE REVIEW

File: flask_ldap3_login/forms.py [hunk 1]
```diff
@@ -25,8 +25,8 @@ class LDAPLoginForm(FlaskForm):
 
     """
 
-    username = wtforms.StringField("Username", validators=[validators.Required()])
-    password = wtforms.PasswordField("Password", validators=[validators.Required()])
+    username = wtforms.StringField("Username", validators=[validators.DataRequired()])
```
Comment (nickw444) [inline flask_ldap3_login/forms.py:28]: Looking into the docs, `DataRequired` appears to be the drop-in replacement. 

Am wondering whether we should migrate to InputRequired instead, as [suggested by the docs](https://wtforms.readthedocs.io/en/stable/validators/#wtforms.validators.DataRequired):

> NOTE this validator used to be called Required but the way it behaved (requiring coerced data, not input data) meant it functioned in a way which was not symmetric to the Optional validator and furthermore caused confusion with certain fields which coerced data to ‘falsey’ values like 0, Decimal(0), time(0) etc. Unless a very specific reason exists, we recommend using the InputRequired instead.

However that probably opens possibility for a regression in some particular use-cases, so happy for this to be done as a drop-in migration.

**Review by nickw444 — APPROVED** [review]

**Comment by shortywz** [comment]
Are we going to get this PR merged, or are we still waiting for a new maintainer?

**Comment by nickw444** [comment]
Merged, yes I can do that. As for releasing it, I'd like to get either @HeMan or @gmacon to advise because as I understand `master` contains unreleased breaking changes which were intended to land in `v1.0.x`.

**Comment by nickw444** [comment]
I think the best way forward (given the slowdown on v1 progress) will be to move `master` to a new branch (`v1`) and roll `master` back to reflect the current released version and apply patches to both branches as they come through. What do you think @gmacon @HeMan ?

**Comment by gmacon** [comment]
I think we should just merge this and then release the current state as v1.0.0. The remaining items look fairly minor even though they are technically backwards-breaking. If we write good release notes, we shouldn't scare people off from taking the breaking changes.

**Review by gmacon — APPROVED** [review]

**Pull request merged**